### ISSUE TITLE
CK - ANC Doctor visit form

### ISSUE THREAD

**Issue by vinayvenu** [OpenCHS/openchs-product#183]
Acceptance Criteria

 As a CK user i should be able to see a ANC Doctor visit form for a woman enrolled in Pregnancy Program
 The fields would be as mentioned in the spreadsheet : https://docs.google.com/spreadsheets/d/1Ni2DioQ1c7XLEKvqzUKP1o32Etnsz5mGTsI_uOWi5K8/edit#gid=1479316303

Notes

 Remove all referral rules from this form. 
 Ignore the comments column on the sheet. 
 Focus on fields that are highlighted. These are the places that have changes from the original form
 Verify the time when form elements should show up based on column A. eg: Fundal Height From Pubic Symphysis is calculated from week 12 based on column A. 
 GA(Gestational Age) needs to be provided by Doctor.
 All rules need to rely on Doctor given GA.
 Blood sugar needs to be split into 2 fields Fasting & Random.

Tech Tasks :

- This form would be an override of ANC form in health module

Issues – 30th July 2018 
1) Difficulty breathing shows up in Trimester1 (should show only after Trimester2) 

2) Fundal Height (and Abdominal Examination section) shows up before 12 weeks 
3) Unclear from document if USG - Viability is mandatory or not (it is Boolean, so okay to be mandatory)
4) Weight <= 35 not being highlighted (there is separate card of weight gain logic)
5) No mention of Assessment and GPAL in document 

6) Verify if USG Diagnostics is relevant here if it is being captured in Lab Test too(irrelevant as it is captured in Lab tests)
